capstone/src/main/webapp/profile-loader.js

```diff
@@ -21,11 +21,16 @@ function getStudentInfo() {
     var profileTitle = document.getElementById('heading');
     profileTitle.innerHTML += studentInfo['name'];
 
-    // Update profile club list
+    // Update profile club list and announcements inbox
     var clubList = document.getElementById('club-content');
+    var inbox = document.getElementById('inbox');
     const clubs = studentInfo['clubs'];
     for(const key in clubs) {
```

Comment:
You can use "of" to just easily get the value

Reply:
Got it, thanks for pointing that out!